### PR TITLE
Kube request

### PR DESCRIPTION
**Pull request by vojta-janousek**

### CODE REVIEW

File: link_shortener/core/initialise_db.py [hunk 1]
```diff
@@ -97,14 +99,24 @@
...
+        )
+    else:
+        app.engine = await create_engine(
+            host='db',
+            port=3306,
+            user='user',
+            password='password',
+            db='db',
+            loop=loop
+        )
```
Comment (radekholy24) [inline link_shortener/core/initialise_db.py:119]: What about having just this:
```suggestion
    app.engine = await create_engine(
        host=config('MYSQL_HOST'),
        port=config('MYSQL_PORT', cast=int),
        user=config('MYSQL_USER'),
        password=config('MYSQL_PASSWORD'),
        db=config('MYSQL_DB'),
        loop=loop
    )
```
? I mean, if people want to run it against localhost, they would set the credentials in `settings.ini` accordingly. But if you prefer this way, I'm fine with that...

File: Documentation/SETTINGS.md [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
 WHITELISTED_EMAIL_1=<email_domain_1>
 WHITELISTED_EMAIL_2=<email_domain_2>
 ...
 WHITELISTED_EMAIL_N=<email_domain_n>
 
+MYSQL_HOST=<mysql_host_name>
+MYSQL_DB=<mysql_db_name>
+MYSQL_USER=<mysql_user_name>
+MYSQL_PASSWORD=<mysql_password>
+MYSQL_ROOT_PASSWORD=<mysql_root_password>
```
Comment (radekholy24) [inline Documentation/SETTINGS.md]: Is this needed anywhere?

File: link_shortener/server.py [hunk 3]
```diff
@@ -68,10 +68,14 @@ def create_app():
...
 
     app.config.OAUTH_PROVIDER = config('OAUTH_PROVIDER')
-    app.config.OAUTH_REDIRECT_URI = config('OAUTH_REDIRECT_URI')
     app.config.OAUTH_SCOPE = config('OAUTH_SCOPE')
     app.config.OAUTH_CLIENT_ID = config('OAUTH_CLIENT_ID')
     app.config.OAUTH_CLIENT_SECRET = config('OAUTH_CLIENT_SECRET')
+    if config('PRODUCTION', default=False, cast=bool):
+        domain = config('DOMAIN_NAME')
+    else:
+        domain = config('LOCAL_HOST')
```
Comment (radekholy24) [inline link_shortener/server.py:77]: Same here... We can have just:
```suggestion
    domain = config('DOMAIN_NAME')
```
and set `DOMAIN_NAME` in `settings.ini` to `http://localhost:8000` if we want to. But if you prefer this way, I'm fine with that...

**Review by radekholy24 — APPROVED** [review]
No serious issues here. Just let me know if you prefer your solution and we can merge it. Thank you!

File: link_shortener/core/initialise_db.py (hunk 1 above)
Comment (vojta-janousek) [reply link_shortener/core/initialise_db.py:119]: It's a DEV/PROD switch for me, so I only have to change one variable in settings.ini between building the image and developing new features.

File: Documentation/SETTINGS.md (hunk 2 above)
Comment (vojta-janousek) [reply Documentation/SETTINGS.md]: For the K8 container to connect to the mysql K8 container. In development, they'll connect through docker-compose.

Comment (radekholy24) [reply Documentation/SETTINGS.md]: Yes, actually I meant this particular one. The `MYSQL_ROOT_PASSWORD` one. Is this used anywhere? The rest is OK.

Comment (vojta-janousek) [reply Documentation/SETTINGS.md]: It should be a mandatory field for db initialisation via docker-compose.

Comment (radekholy24) [reply Documentation/SETTINGS.md]: Yes. Right. As an environment variable. But does it make sense here in `settings.ini`?

Comment (vojta-janousek) [reply Documentation/SETTINGS.md]: Not for us, but perhaps someone else might deploy the app via Docker, and they would've to hide the values in .env/.ini file.
(Removed)

**Review by radekholy24 — APPROVED** [review]
Looks good to me. I will wait if some other changes will be needed because of the GitLab's MR. Otherwise I am ready to merge it.

**Pull request merged**